### PR TITLE
Update unsupported browser affordance

### PR DESCRIPTION
**Pull request by GordyD**
- Change BrowerWarningOverlay into its own page
 - Remove any reference/usage in app.js of BrowserWarningOverlay
 - Update styling and content of the browser warning affordance
 - Move logic for handling when affordance is shown into routing onEnter functions
 - Test onEnter function

### CODE REVIEW

**Comment by GordyD** [comment]
Now the browser warning only shows up when navigated to /patients and its sub-routes.

File: app/routes.js [hunk 1]
```diff
@@ -243,14 +262,15 @@ export const getRoutes = (appContext, store) => {
       <Route path='signup' component={Signup} onEnter={requireNoAuth(api)} />
       <Route path='email-verification' component={EmailVerification} onEnter={requireNotVerified(api, store)} />
       <Route path='profile' component={UserProfile} onEnter={requireAuth(api, store)} />
-      <Route path='patients' component={Patients} onEnter={requireAuth(api, store)} />
-      <Route path='patients/new' component={PatientNew} onEnter={requireAuthAndNoPatient(api, store)} />
-      <Route path='patients/:id/profile' component={PatientProfile} onEnter={requireAuth(api, store)} />
-      <Route path='patients/:id/share' component={Share} onEnter={requireAuth(api, store)} />
-      <Route path='patients/:id/data' component={PatientData} onEnter={requireAuth(api, store)} />
+      <Route path='patients' component={Patients} onEnter={requiresChrome(utils, requireAuth(api, store))} />
```
Comment (GordyD) [inline app/routes.js]: @jebeck here are the commits you were wondering about.

File: app/pages/browserwarning/browserwarning.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    );
+  },
+
+  copyText: function() {
+    var copyText = document.querySelector('.blip-link-text');
+    copyText.select();
+
+    try {
+      var copyCmd = document.execCommand('copy');
+      copyText.setSelectionRange(0,0);
```
Comment (krystophv) [inline app/pages/browserwarning/browserwarning.js]: Could there be a way here to surface to the user whether or not the copy command was successful?

**Comment by krystophv** [comment]
Not sure if it's within the scope of this PR, but in researching I discovered that Microsoft Edge is going to pass the `utils.isChrome` check since it's user agent contains `Chrome` [[source]](https://msdn.microsoft.com/en-us/library/hh869301(v=vs.85).aspx). In the document they specifically call out people using UA sniffing.

**Comment by krystophv** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;Sgr8rlNa&#x2F;7-ux-1-dev-1-tst-1-blip-marcomm-messaging-around-chrome-browser-requirement-and-browser-defaults"> &lt;ux:1, dev:1, tst:1&gt; [blip] [marcomm] messaging around chrome browser requirement and browser defaults</a>

**Comment by krystophv** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;qCl1dejO&#x2F;8-ux-1-dev-2-tst-2-signup-account-creation-as-a-user-i-want-to-be-able-to-begin-the-signup-process-in-browsers-that-are-not-chrome"> &lt;ux:1, dev:2, tst:2&gt; [signup] [account creation] as a user, i want to be able to begin the signup process in browsers that are not chrome</a>

**Comment by GordyD** [comment]
@krystophv @jebeck This ticket has been updated to deal with *just* https://trello.com/c/Sgr8rlNa/7-ux-1-dev-1-tst-1-blip-marcomm-messaging-around-chrome-browser-requirement-and-browser-defaults with the addition of:

 - only show download text for chrome if we can detect the user is on a non-mobile device (e.g. a desktop or laptop)

I will break out a second branch to deal with the other ticket now.

**Comment by GordyD** [comment]
Following a meeting with Sara, Brandon and Howard this PR has been updated to only lock down patients/:id/data URL with a browser warning. All other pages are accessible now.

**Pull request merged**